CitationStyleLanguagePlugin.inc.php

```diff
@@ -335,24 +391,22 @@ public function getArticleTemplateData($hookName, $args)
      * @see Mendeley's mappings http://support.mendeley.com/customer/portal/articles/364144-csl-type-mapping
      *
      * @param Request $request
-     * @param Submission $article
+     * @param Submission $submission
      * @param string $citationStyle Name of the citation style to use.
      * @param Issue $issue Optional. Will fetch from db if not passed.
      * @param Publication $publication Optional. A particular version
      *
      * @return string
      */
-    public function getCitation($request, $article, $citationStyle = 'apa', $issue = null, $publication = null)
-    {
-        $publication ??= $article->getCurrentPublication();
-        $issue ??= Repo::issue()->get($publication->getData('issueId'));
+    public function getCitation($request, $submission, $citationStyle = 'apa', $issue = null) {
```

Comment:
Did you intend to remove `$publication = null` from the function signature here?

Reply:
I removed mistakenly removed it, but restored it the [last commit](https://github.com/pkp/citationStyleLanguage/pull/90/commits/70b5db888799c3b0da5a1240854f2e7bd5de2af0).